src/wp-includes/media.php

```diff
@@ -5506,6 +5506,16 @@ function wp_get_loading_attr_default( $context ) {
 		return false;
 	}
 
+	/*
+	 * Skip programmatically created images within post content as they need to be handled together with the other
+	 * images within the post content.
+	 * Without this clause, they would already be counted below which skews the number and can result in the first
+	 * post content image being lazy-loaded only because there are images elsewhere in the post content.
+	 */
+	if ( ( 'the_post_thumbnail' === $context || 'wp_get_attachment_image' === $context ) && doing_filter( 'the_content' ) ) {
```

Comment:
As these same values are used here, I think this should be an array  - https://github.com/WordPress/wordpress-develop/pull/4424/files#diff-4d28b6652f0dab208c19aa45e1cf09f3c967060d8cc77502bfcd473a3216b612R5465

Reply:
I'm not sure I understand your comment ? Used where ? And why does that mean it needs to be an array ?

Reply:
Doesn't answer my questions.

Reply:
Oh, duh! I didn't realize you were linking to another PR as the code view just showed me the same file... 🤦🏻‍♀️ 

Having said that... If there are only two values, creating a use-only-once array is 100% overkill and `in_array()` will not help performance. So I still don't see your point.

Reply:
@spacedmonkey This was intentionally reverted in https://github.com/WordPress/wordpress-develop/pull/4300/commits/a2096b96de15aaafa5c48b26312883e5e1d5f365, in response to https://github.com/WordPress/wordpress-develop/pull/4300#discussion_r1196900653.

I think we should stop this nit-picking, based on that this was already heavily discussed before. We're running in circles here.